src/machine/machine_mimxrt1062.go

```diff
@@ -17,29 +19,56 @@ type PinMode uint8
 
 const (
 	// GPIO
-	PinInput PinMode = iota
-	PinInputPullUp
-	PinInputPullDown
-	PinOutput
-	PinOutputOpenDrain
-	PinDisable
+	PinInput           PinMode = 0
+	PinInputPullUp     PinMode = 1
```

Comment:
Are these changes intentional? The code looks cleaner to me as it was before (with `iota`).

Reply:
They were intentional, but only because I was trying to match the convention from SAMD51. Looks like SAMD21 and STM32 do the same (see: `machine_atsamd21.go`, `machine_atsamd51.go`, and `machine_stm32_moder_gpio.go`)

I also found it useful to be able to immediately look up the integer value of these modes when debugging.

Now that these aren't being debugged anymore, there isn't really a benefit. 

I will revert these back to using `iota`.